src/components/Functionality.js

```diff
@@ -1,43 +1,83 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './Calculator.css';
+import calculate from '../logic/calculate';
 
-const Functionality = () => (
-  <div className="main">
-    <div className="calculator">
-      <div className="screen">
-        <input type="text" value="0" className="silver" />
-      </div>
-      <div className="row first-row">
-        <input value="AC" type="button" />
-        <input value="+/-" type="button" />
-        <input value="%" type="button" />
-        <input value="/" type="button" className="color" />
-      </div>
-      <div className="row second-row">
-        <input value="7" type="button" />
-        <input value="8" type="button" />
-        <input value="9" type="button" />
-        <input value="X" type="button" className="color" />
-      </div>
-      <div className="row third-row">
-        <input value="4" type="button" />
-        <input value="5" type="button" />
-        <input value="6" type="button" />
-        <input value="-" type="button" className="color" />
-      </div>
-      <div className="row fourth-row">
-        <input value="1" type="button" />
-        <input value="2" type="button" />
-        <input value="3" type="button" />
-        <input value="+" type="button" className="color" />
-      </div>
-      <div className="row fifth-row">
-        <input value="0" type="button" className="expand" />
-        <input value="." type="button" />
-        <input value="=" type="button" className="color" />
+const Functionality = () => {
```

Comment:
- [ ] This component is a child component of the Calculator component. But there are no props being passed from Calculator to this one - this makes Calculator completely useless. Kindly refactor your code so that Calculator handles the state and you pass some kind of props to this child component to do something.

  ![image](https://user-images.githubusercontent.com/81468675/226874124-57254f4d-4f54-45aa-850f-2e90d707e212.png)

Reply:
Hello @julie-ify can you please check your slack messages, I messaged you there explaining why I did it this way.